flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/DefaultConfigurableOptionsFactory.java

```diff
@@ -156,6 +159,19 @@ public ColumnFamilyOptions createColumnOptions(
             blockBasedTableConfig.setBlockCacheSize(getBlockCacheSize());
         }
 
+        if (isOptionConfigured(USE_BLOOM_FILTER)) {
+            final boolean enabled = Boolean.parseBoolean(getInternal(USE_BLOOM_FILTER.key()));
+            if (enabled) {
+                final double bitsPerKey =
+                        isOptionConfigured(BLOOM_FILTER_BITS_PER_KEY)
+                                ? Double.parseDouble(getInternal(BLOOM_FILTER_BITS_PER_KEY.key()))
+                                : BLOOM_FILTER_BITS_PER_KEY.defaultValue();
+                BloomFilter bloomFilter = new BloomFilter(bitsPerKey);
```

Comment:
I think full format of bloom filter behaves betterr than old block-based format, you could add another option to describe whether the filter is block format (default value could be full format).